gulpfile.js

```diff
@@ -64,6 +55,11 @@ gulp.task('css', () => {
     .pipe(gulp.dest('dist/public/stylesheets/'));
 });
 
+gulp.task('vendor-assets', () => {
+  gulp.src([
+    'node_modules/govuk-elements-sass/node_modules/govuk_frontend_toolkit/images/*.png',
+  ]).pipe(gulp.dest('vendor/govuk_frontend_toolkit/assets/images'));
```

Comment:
Want to still keep it separate because if it was an npm module we could include it would be.  The only reason its checked in is because it has to be downloaded from here:
http://alphagov.github.io/govuk_template/  (See Mustache inheritance link) which is the compiled version.